src/Components/Shared/test/TestRenderer.cs

```diff
@@ -40,12 +40,17 @@ public new Task RenderRootComponentAsync(int componentId)
         public new Task RenderRootComponentAsync(int componentId, ParameterCollection parameters)
             => InvokeAsync(() => base.RenderRootComponentAsync(componentId, parameters));
 
-        public new Task DispatchEventAsync(int componentId, int eventHandlerId, UIEventArgs eventArgs)
-            => InvokeAsync(() => base.DispatchEventAsync(componentId, eventHandlerId, eventArgs));
+        public new Task DispatchEventAsync(int eventHandlerId, UIEventArgs args)
+            => InvokeAsync(() => base.DispatchEventAsync(eventHandlerId, args));
 
         public T InstantiateComponent<T>() where T : IComponent
             => (T)InstantiateComponent(typeof(T));
 
+        public Task WaitForAsyncWork()
```

Comment:
This doesn't seem to be used. Can it be removed?

Reply:
done